deployment/docker-compose/template.env

```diff
@@ -42,8 +42,10 @@ NEXT_PUBLIC_BACKEND_URL="https://localhost/api"
 HUGGINGFACE_MODEL="thenlper/gte-large"
 EMBEDDINGS_ENDPOINT="http://embeddings:8080"
 EMBEDDINGS_API_KEY="embeddings"
-# endpoint for the backend to call - must be set to the internal service name and port
```

Comment:
We should keep the comment: "must be set to the internal service name and port". I don't think this is common knowledge